openquakeplatform_ipt/views.py

```diff
@@ -769,26 +804,32 @@ def event_based_prepare(request, **kwargs):
     jobini += "truncation_level = %s\n" % data['truncation_level']
     jobini += "maximum_distance = %s\n" % data['maximum_distance']
     jobini += "investigation_time = %s\n" % data['investigation_time']
-    jobini += "ses_per_logic_tree_path = %s\n" % data['ses_per_logic_tree_path']
-    jobini += "number_of_logic_tree_samples = %s\n" % data['number_of_logic_tree_samples']
-    jobini += "ground_motion_correlation_model = %s\n" % data['ground_motion_correlation_model']
+    jobini += ("ses_per_logic_tree_path = %s\n" %
+               data['ses_per_logic_tree_path'])
+    jobini += ("number_of_logic_tree_samples = %s\n" %
+               data['number_of_logic_tree_samples'])
+    jobini += ("ground_motion_correlation_model = %s\n" %
+               data['ground_motion_correlation_model'])
     if data['ground_motion_correlation_model'] == 'JB2009':
-        jobini += "ground_motion_correlation_params = {\"vs30_clustering\": True}"
+        jobini += ('ground_motion_correlation_params = '
```

Comment:
Switching between string delimiters increase the difficult to match things consistenty with the rest of the code. Please revert to double quote as delimiter.